src/v3/test/integration/__snapshots__/authenticator-apple-sso-extension.test.tsx.snap

```diff
@@ -41,7 +41,7 @@ exports[`authenticator-apple-sso-extension calls verify then cancels the transac
                     class="MuiBox-root emotion-7"
                   >
                     <h2
-                      class="MuiTypography-root MuiTypography-h3 emotion-8"
+                      class="MuiTypography-root MuiTypography-h4 emotion-8"
```

Comment:
Are these MUI class changes intended?

Reply:
Yep these are intentional, what seemed to have happened is that this file was never updated after the Title variant change was made (from h3 to h4 by Alex). I am unsure as to how it wasnt caught until now though 😓